src/other_functions.jl

```diff
@@ -126,3 +129,50 @@ function abs{T<:Real}(a::Taylor1{TaylorN{T}})
         (abs(x) is not differentiable at x=0)."""))
     end
 end
+"""
+    abs(a)
+
+Returns `a` if `constant_term(a) > 0` and `-a` if `constant_term(a) < 0` for
+`a <:Union{Taylor1,TaylorN}`.
+Notice that `typeof(abs(a)) <: AbstractSeries`.
+"""abs
+
+
+#norm
+"""
+    norm(x::AbstractSeries,p::Real)
+
+Computes the p-norm of an `AbstractSeries` defined by $P(\vec{x}) = \sum_k a_k \vec{x}^k$ as
```

Comment:
Should the `$` signs be escaped?

Reply:
What seems to work best, at least locally, is to use `doc` before the opening `"""` quotes, and then write the math expression as ``` ``P(\vec{x}) = \sum_k a_k \vec{x}^k`` ```